queue/src/fiskaltrust.Middleware.Queue/ReceiptRequestHelper.cs

```diff
@@ -178,7 +178,10 @@ public static string GetCountry(ReceiptRequest data)
             {
                 0x4445000000000000 => "DE",
                 0x4652000000000000 => "FR",
-                _ => "AT",
+                0x4154000000000000 => "AT",
+                0x4D45000000000000 => "ME",
+                0x4954000000000000 => "IT",
```

Comment:
Does this mean that we didn't really have support for IT in yet? 

Also the fallback with AT is something that we should take a closer look at. Do you think having the DEFAULT fallback could cause troubles @TSchmiedlechner ?

Reply:
Yes. we should definitely add IT in the main branch as well before we release the next version 😅 This method is only used here https://github.com/fiskaltrust/middleware/blob/2c285afbc67667ab87738c5683114af5f6478d89/queue/src/fiskaltrust.Middleware.Queue/SignProcessor.cs#L137 and `ftQueueItem.country` is not read anywhere in the MW (maybe in the portal?)

Reply:
Okay so this would be some sort of breaking change if we add it this way. All folks that are sending receiptcases that are "wrong" have gotten AT so far (which is not really correct but fair enough). In the future they would get "DEFAULT". I guess we should probably leave AT as defaut. Probably @TSchmiedlechner can add his two cents here since he has been more involved in implementations than me.